.circleci/config.yml

```diff
@@ -0,0 +1,12 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+# Invoke jobs via workflows
+# See: https://circleci.com/docs/2.0/configuration-reference/#workflows
+workflows:
```

Comment:
add this code above workflows and after version, 

`
orbs:
  slack: circleci/slack@4.3.3
jobs:
  build:
    working_directory: ~/repo
    docker:
      - image: circleci/node:10.16.3
    steps:
      - checkout
      - run:
          name: Update NPM
          command: "sudo npm install -g npm@5"
      - restore_cache:
          key: dependency-cache-{{ checksum "package-lock.json" }}
      - run:
          name: Install Dependencies
          command: npm install
      - save_cache:
          key: dependency-cache-{{ checksum "package-lock.json" }}
          paths:
            - ./node_modules
      - run:
          name: Build app
          command: npm run build
      - slack/notify:
          event: fail
          mentions: '@channel'
          template: basic_fail_1
      - slack/notify:
          event: pass
          template: success_tagged_deploy_1
`

you should adapt it to build the vite app

Reply:
Done. I'm not really sure how to divide the code by line. I look up online and use some config sample file on the circle CI docs as reference. 
![image](https://user-images.githubusercontent.com/109154865/219701589-21b4164c-03e5-4ba2-8b60-5a66746715d3.png)

Reply:
Hi @Garnetuwu 

Please update the docket image used in ci to this one: `cimg/node:19.7.0` 
I have a strong feeling this will fix the ci step that fails. 

Basically we are using currently a deprecated image, and we get a warning in circle ci that tells us this and to replace with newer one that are namespaced with `cimg` 

The above one is the latest version so I think it should work fine for our purposes